libs/openFrameworks/types/ofBaseTypes.cpp

```diff
@@ -48,6 +48,11 @@ ofBaseVideoPlayer::~ofBaseVideoPlayer(){
 
 }
 
+void ofBaseVideoPlayer::loadAsync(string name){
+	ofLogWarning("ofBaseVideoPlayer") << "loadAsync() not implemented";
```

Comment:
Done



> Den 20/02/2015 kl. 16.07 skrev arturo <notifications@github.com>:
> 
> In libs/openFrameworks/types/ofBaseTypes.cpp:
> 
> > @@ -48,6 +48,11 @@ ofBaseVideoPlayer::~ofBaseVideoPlayer(){
> >  
> >  }
> >  
> > +void ofBaseVideoPlayer::loadAsync(string name){
> > +	ofLogWarning("ofBaseVideoPlayer") << "loadAsync() not implemented";
> super minor thing but can you change this to "loadAsync() not implemented, loading synchronously";
> 
> otherwise it might seem that it has just failed.
> 
> i'll merge it afterwards.
> 
> —
> Reply to this email directly or view it on GitHub.
>